src/CoreBundle/private/modules/MTHeader/MTHeader.class.php

```diff
@@ -450,16 +451,29 @@ public function ClearCache()
      */
     protected function ClearOutBox($sDir)
     {
-        $fileManager = $this->getFileManager();
-
         if (false === is_dir(PATH_OUTBOX)) {
             return;
         }
         $sDir = PATH_OUTBOX.$sDir;
         if ('/' !== $sDir[strlen($sDir) - 1]) {
```

Comment:
Currently this is redundant as it is duplicated using rtrim in clearDir.

Reply:
ok